app/src/main/java/org/sopt/dosopttemplate/MainActivity.kt

```diff
@@ -1,11 +1,67 @@
 package org.sopt.dosopttemplate
 
-import androidx.appcompat.app.AppCompatActivity
+
+import android.content.Context
+import android.content.Intent
+import android.content.SharedPreferences
 import android.os.Bundle
+import android.widget.Toast
+import androidx.activity.OnBackPressedCallback
+import androidx.appcompat.app.AppCompatActivity
+import org.sopt.dosopttemplate.databinding.ActivityMainBinding
 
 class MainActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityMainBinding
+    private lateinit var userInfo: UserInfo
+    private lateinit var sharedPreferences: SharedPreferences
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        binding = ActivityMainBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        userInfo = getUserInfo()
+
+        binding.tvMainViewid.text = "${userInfo.userId}"
```

Comment:
```suggestion
        binding.tvMainViewid.text = userInfo.userId
```
이미 userId가 string 타입이니깐 요렇게 작성 가능!! 아래 변수들도 동일합니다

Reply:
오호 감사합니당